contrib/docs-code-samples/expand-api-display-access/01-expand-beach/curl.sh

```diff
@@ -6,5 +6,5 @@ curl -G --silent \
      --data-urlencode "relation=access" \
      --data-urlencode "object=/photos/beach.jpg" \
      --data-urlencode "max-depth=3" \
-     http://127.0.0.1:4466/expand | \
+     http://127.0.0.1:4466/acl/expand | \
```

Comment:
ACL for Access Control List? Is that the nomenclautur you want to go here?

Reply:
I was also not sure about that, but basically yes. The gRPC API is also `package ory.keto.acl.v1alpha1`, so it would be consistent.
In some way you could say that relation tuples are an ACL?

Reply:
Later on we can have `/rbac/check` and others.